avionics/common/src/avionics.cpp

```diff
@@ -282,22 +180,23 @@ int main(void) {
             old_time = new_time;
 
             sensors.poll(timestamp);
+            new_time_int =
+                static_cast<uint32_t>(timestamp.time_since_epoch().count());
```

Comment:
I would rename this to something like sensor_poll_time to avoid confusion like we had before.

Reply:
actually wait - it might be better to instead put a `last_poll_time_` variable in `SensorCollection`, since that would be even clearer with regards to what's going on with the timestamp. But doing that would deserve it's own pull request - can I ask to defer this?

Reply:
See #138